src/react-textarea-count.jsx

```diff
@@ -26,7 +27,7 @@ export default class Textarea extends Component {
 
   renderCount() {
     const { countLimit } = this.props;
-    const className = `textarea__count${this.textarea && countLimit && this.textarea.value.length > countLimit ? ' textarea__count--error' : ''}`;
+    const className = `textarea__count${this.textarea && countLimit && this.textarea.value.length >= countLimit ? ' textarea__count--error' : ''}`;
```

Comment:
I don't think this change belongs in this pull request.

If anything, it should be extracted out to a separate request. (And I would debate whether it should be accepted, because if the limit is 100 and there are 100 characters, that should be valid, shouldn't it?)